chasm/src/main/java/org/quiltmc/chasm/internal/TransformerSorter.java

```diff
@@ -25,9 +29,31 @@ public static List<List<Transformer>> sort(Collection<Transformer> transformers)
             infoById.put(transformer.getId(), vertex);
         }
 
+        // Create vertices for transformers that don't exist.
+        {
+            Set<String> knownIds = new HashSet<>(infoById.keySet());
+            transformers.stream()
+                    .flatMap(transformer -> Stream.concat(
+                            Stream.concat(
+                                    transformer.mustRunAfter(knownIds).stream(),
+                                    transformer.mustRunBefore(knownIds).stream()
+                            ),
+                            Stream.concat(
+                                    transformer.mustRunRoundAfter(knownIds).stream(),
+                                    transformer.mustRunBefore(knownIds).stream()
+                            )
+                    ))
+                    .distinct()
+                    .filter(s -> !infoById.containsKey(s))
+                    .forEach(s -> infoById.put(s, new TransformerInfo(null)));
```

Comment:
I'm tempted to turn this into a regular loop, with how nothing else in the method uses Streams.

Reply:
I approve of using a normal loop :)